### PR TITLE
Some optionals tests

### PR DESCRIPTION
**Pull request by cjllanwarne**

### CODE REVIEW

File: src/main/resources/standardTestCases/optional_parameter/optional_parameter.inputs [hunk 1]
```diff
@@ -1,4 +1,7 @@
 {
-    "default.hello.person": "alice"
+    "optional_parameter.select_first_with_optionals.wfSupplied": "wfSupplied",
+    "optional_parameter.interpolation_additions.wfSupplied": "wfSupplied",
+    "optional_parameter.default_interpolations.wfSupplied": "wfSupplied"
 }
 
+in
```
Comment (cjllanwarne) [inline src/main/resources/standardTestCases/optional_parameter/optional_parameter.inputs]: Delete this floater. Add a newline

File: src/main/resources/standardTestCases/optional_parameter/optional_parameter.wdl [hunk 2]
```diff
@@ -1,35 +1,73 @@
...
+		echo ${"supplied: " + supplied}
+		echo ${"suppliedInt: " + suppliedInt}
+		echo ${"wfSupplied: " + wfSupplied}
+		echo ${"unsupplied: " + unsupplied}
+		echo ${"unsuppliedInt: " + unsuppliedInt}
+	}
+	runtime {
+      docker: "ubuntu:latest"
+    }
+	output {
```
Comment (cjllanwarne) [inline src/main/resources/standardTestCases/optional_parameter/optional_parameter.wdl:41]: Spacing

File: src/main/resources/standardTestCases/select_functions/select_functions.wdl [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    Array[String] allFromMix = select_all([makeStringTrue.out, makeStringFalse.out])
+    String firstFromTruths = select_first([makeStringTrue.out, makeStringTrue.out])
+    String firstFromFalses = select_first([makeStringFalse.out, makeStringFalse.out, "default"])
+    String firstFromMix = select_first([makeStringFalse.out, makeStringTrue.out])
+    String? mst = makeStringTrue.out
+    Array[String?] msts = [mst, mst]
+    Array[String] allFromMsts = select_all(msts)
+    String firstFromMsts = select_first(msts)
+  }
+}
```
Comment (cjllanwarne) [inline src/main/resources/standardTestCases/select_functions/select_functions.wdl]: Newline

**Comment by mcovarr** [comment]
:+1:

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/137/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)

**Comment by Horneth** [comment]
👍 

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/137/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)

**Pull request merged**